lib/src/header_parser/type_extractor/cxtypekindmap.dart

```diff
@@ -27,6 +28,22 @@ var cxTypeKindToSupportedNativeTypes = <int, SupportedNativeType>{
   clang.CXTypeKind.CXType_Char_U: SupportedNativeType.Uint8,
 };
 
+var cxTypeKindToImportedTypes = <String, ImportedType>{
+  'void': voidType,
+  'unsigned char': unsignedCharType,
+  'char': charType,
+  'unsigned short': unsignedShortType,
+  'short': shortType,
+  'unsigned int': unsignedIntType,
+  'int': intType,
+  'unsigned long': unsignedLongType,
```

Comment:
long and unsigned long do not have a fixed size type on the various architectures

(unsigned) char, (unsigned) short, (unsigned) int, and (unsigned) long long do. However, for these we should probably also target the typedefs in `package:ffi` for clarity. (The C standard says at least 8 bytes for long long, so we could even have platforms in the future which decide for another size. In that case we can just update `package:ffi` and don't have to update `package:ffigen`.)

Reply:
Those were the defaults before this PR. Will update these once package:ffi has a dev release.

Reply:
Updated